pkgs/data/fonts/victor-mono/default.nix

```diff
@@ -1,14 +1,10 @@
-{ lib, fetchFromGitHub }:
+{ lib, fetchzip }:
 
 let
-  pname = "victor-mono";
-  version = "1.4.1";
-in fetchFromGitHub rec {
-  name = "${pname}-${version}";
-
-  owner = "rubjo";
-  repo = pname;
-  rev = "v${version}";
+  version = "1.5.0";
+in
+fetchzip {
```

Comment:
Could you provide some motivation for reverting to `fetchzip` instead of `fetchFromGitHub`?

The latter is more declarative and may quickly accommodate new URL format if GttHub someday decides to change its endpoint for files. Also, it's a bit out of scope for a simple package update :)

Reply:
Since the fetchers are fixed output derivations this will only be a problem if there is no cache.